hangman_lunch.rb

```diff
@@ -0,0 +1,161 @@
+#Lila and Corinne
+
+# works through lunch, first req of dinner (not penalizing repeat guesses on same letter)
+
+require "colorize"
+
+class Hangman
+	def initialize
+
+		# library
+		@words = ["orca", "cheese", "cake", "pumpkin", "sun", "apple", "cloud", "paint", "bird", "travel", "lollipop", "ada"]
+
+		@answer_word = choose_word # array of characters
+		@answer_copy = @answer_word.dup
+		@word_display = @answer_copy.fill("_ ")
+		@guessed_letters = [] # guessed letter array
+
+		@default_guesses = 7
+		@wrong_count = 0
+
+		#creates
+		start
+	end
+
+# Display (art, word)
+
+
+	def start
+		puts "Let's play HANGMAN."
+		update_visuals
+		@keep_playing = true
+
+		play
+	end
+
+	def play
+		# checks for loss due to use of all guesses
+		while @keep_playing == true
+			if @wrong_count < 7 && win != true
+				puts "Guess one letter and then hit enter. Type \"quit\" if you don't want to play."
+				print "YOUR GUESS: "
+				@current_guess = get_guess #gets guess
```

Comment:
Based on the way the guess is being returned from `get_guess` and passed into `check_guess` you should probably just use a local variable here rather than an instance variable